src/styles/ant-variables.less

```diff
@@ -0,0 +1,65 @@
+@import '../../node_modules/antd/lib/style/colors/colorPalette';
+
+// -------- Colors -----------
+@primary-color : #1890ff;
+@info-color : #1890ff;
+@success-color : #52c41a;
+@processing-color : #1890ff;
+@error-color : #f5222d;
+@highlight-color : #f5222d;
+@warning-color : #faad14;
+@normal-color : #d9d9d9;
+@white : #fff;
+@black : #000;
+
+// Background color for `<body>`
+@body-background : #fff;
```

Comment:
Lots of these aren't necessarily needed but I left them in for the sake of us knowing that they exist